data/nmstate/003-operator.yaml

```diff
@@ -39,78 +40,74 @@ spec:
           env:
             - name: WATCH_NAMESPACE
               value: ""
-            - name: RUN_WEBHOOK_SERVER
-              value: ""
             - name: POD_NAME
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: RUN_WEBHOOK_SERVER
+              value: ""
             - name: OPERATOR_NAME
-              value: "{{template "handlerPrefix" .}}nmstate-handler"
-            - name: NODE_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: spec.nodeName
-            - name: INTERFACES_FILTER
-              valueFrom:
-                configMapKeyRef:
-                  name: {{template "handlerPrefix" .}}nmstate-config
-                  key: interfaces_filter
-            - name: WEBHOOK_PORT
-              value: "54874"
+              value: "{{template "handlerPrefix" .}}nmstate"
             - name: ENABLE_PROFILER
               value: "False"
             - name: PROFILER_PORT
               value: "6060"
-            - name: NMSTATE_INSTANCE_NODE_LOCK_FILE
-              value: "/var/k8s_nmstate/handler_lock"
+          ports:
+          - containerPort: 8443
+            name: webhook-server
+            protocol: TCP
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: webhook-server
+              scheme: HTTPS
+              httpHeaders:
+              - name: Content-Type
+                value: application/json
+            initialDelaySeconds: 10
+            periodSeconds: 10
           volumeMounts:
-          - name: dbus-socket
-            mountPath: /run/dbus/system_bus_socket
-          - name: nmstate-lock
-            mountPath: /var/k8s_nmstate
-          securityContext:
-            privileged: true
+          - name: tls-key-pair
+            readOnly: true
+            mountPath: /etc/webhook/certs
       volumes:
-      - name: dbus-socket
-        hostPath:
-          path: /run/dbus/system_bus_socket
-          type: Socket
-      - name: nmstate-lock
-        hostPath:
-          path: /var/k8s_nmstate
+        - name: tls-key-pair
+          secret:
+            secretName: {{template "handlerPrefix" .}}nmstate-webhook
```

Comment:
do we have support for this templating on CNAO?

Reply:
CNAO is already dealing with it at master.